api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -28,23 +22,25 @@
 )
 from opentrons.protocols.context.instrument import AbstractInstrument
 from opentrons.protocols.api_support.types import APIVersion
-from .labware import Labware, OutOfTipsError, Well, next_available_tip
+
+# from .labware import Labware, OutOfTipsError, Well
+from opentrons.protocol_api import labware
```

Comment:
Prefer local imports from the same level

```suggestion
from . import labware
```